packages/testing/ts-jest/src/mocks.spec.ts

```diff
@@ -226,6 +226,13 @@ describe('Mocks', () => {
       expect(mock.toString()).toEqual('[object Object]');
       expect(mock.nested.toString()).toEqual('function () { [native code] }');
     });
+
+    it('nested properties should equal its partial', () => {
+      const mock = createMock<any>({ foo: { bar: 1 } });
```

Comment:
@rbnayax Is this enough to prove that primitives are built as expected?
Does this test run if you don't add your change, i'd like to make sure it does enough

Reply:
I verified that it failed on the previous code.  
The previous PR introduced new tests to verify the primitive behaviour and they are passing after my change so I believe that is enough